README.adoc

```diff
@@ -193,28 +189,31 @@ system/pom.xml
+After stopping the `query` service, restart it with the `development` config profile. To do so, run the following Maven goal from the `query` directory:
 
 [role='command']
 ```
 mvn liberty:dev -Dliberty.var.mp.config.profile="development"
 ```
 
-When you set `mp.config.profile` to `development` as a Liberty variable, each `%development.*` property overrides the value of its original property. For example, the [hotspot=dev.port file=0]`%development.system.httpPort` property overrides the [hotspot=port file=0]`system.httpPort` property and the value is resolved to `9081`.
+By setting `mp.config.profile` to `development` as a Liberty variable, each `%development.*` property will override the value of its original property. For example, the [hotspot=dev.port file=0]`%development.system.httpPort` property will override the [hotspot=port file=0]`system.httpPort` property and the value will be resolved to `9081` in this case.
 
-The `mp.config.profile` property can be set in any of the ConfigSources. However, it is read only at application startup. Any changes after that are ignored until the next application startup.
+Keep in mind that the `mp.config.profile` property can be set in any of the ConfigSources, but it is read only at application startup. Any changes made after that will be ignored until the next application startup.
 
-Now, you can access the application by the http://localhost:9085/query/systems/localhost URL. You can see the current OS and Java version in JSON format.
+After restarting the `query` service with the `development` config profile, you can now access the application by using the http://localhost:9085/query/systems/localhost URL. You can see the current OS and Java version in JSON format.
 
 // =================================================================================================
 == Creating the configuration profile using ConfigSource
 
-You can also provide multiple ConfigSources with the names formatted as `microprofile-config-<mp.config.profile>.properties` under the `META-INF` directory.
+In addition to creating configuration profiles at the property level, it's also possible to create configuration profiles using ConfigSources. This can be particularly useful when you need to manage a large number of configuration properties for different environments. For instance, each DevOps stage may require the application to run on different ports and context roots. Using ConfigSource to externalize and customize these configurations leads to a cleaner and more modular code design.
+
+To create a configuration profile at the Config Source level, simply create a file under the `META-INF` folder on the classpath with the naming convention `microprofile-config-<mp.config.profile>.properties`. Once you have created the file, you can add your configuration properties to it with the `<property name>=<value>` syntax. 
```

Comment:
here again using mp.config.profile in the example syntax seems potentially confusing. Whatever you end up using for the earlier instance should be reused here, possibly

`microprofile-config-<config_profile_id>.properties`

Reply:
Edited the change requested in https://github.com/OpenLiberty/draft-guide-microprofile-config-profile/pull/14/commits/1fb7c4965bdfe9a65e7ada76a6b52ab381be86ea